prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Story {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  content   String
+  postal    String
+  email     String?
+  twitter   String?
+  phone     String?
+  approved  Boolean  @default(false)
+  viewCount Int      @default(0)
+}
```

Comment:
```suggestion
    @@index([postal, approved])
}
```

Since we'll probably need to filter on location and we're already filtering on approved